src/Digdir.Domain.Dialogporten.Application/Digdir.Domain.Dialogporten.Application.csproj

```diff
@@ -8,16 +8,19 @@
         <!-- In order to support doc blocks for Swagger generation in PatchDialogsController.cs -->
         <GenerateDocumentationFile>true</GenerateDocumentationFile>
         <NoWarn>1591</NoWarn> <!-- Disable warnings for missing XML comments -->
-    </PropertyGroup>
+      <UserSecretsId>41944251-f40a-41f8-9730-d6d105af39ed</UserSecretsId>
```

Comment:
Her lager du en splitter ny user secrets mappe på et prosjekt som ikke er ment til å være en runtime. Anbefaler at du følger mønsteret som er lagt opp for de andre runtimene våre 🙂

Reply:
Skal få gjenbrukt eksisterende secretsid.